core/modules/user/user.module

```diff
@@ -759,6 +767,22 @@ function user_account_form(&$form, &$form_state) {
   $form['#validate'][] = 'user_account_form_validate';
   module_load_include('password.inc', 'user', 'user');
   $reject_weak = user_password_reject_weak($user->name);
+  // Label the threshold for the password description.
+  $strength_threshold = $site_config->get('user_password_strength_threshold');
+  switch (TRUE) {
```

Comment:
Shouldn't this be `switch ($strength_threshold)` instead?

Reply:
Nope.

It's the only way to do the comparison in each case statement.

Reply:
@herbdool I'd prefer if you find a different way to accomplish that. While `switch (TRUE)` is valid per se, it's *very* uncommon in Backdrop and it will baffle the next developer who's touching this.

Maybe you're not a fan of `if ... elseif`, but in this case it seems more comprehensible to me.

Reply:
@indigoxela I didn't see any other examples in Backdrop core either so to be consistent I can change it. With something flat like this `if ... elseif` is okay by me. I mostly object to where there are multiple nested levels of `if` and `if ... elseif`.